paimon-flink/paimon-flink-common/src/main/resources/META-INF/services/org.apache.paimon.factories.Factory

```diff
@@ -42,4 +42,4 @@ org.apache.paimon.flink.procedure.MigrateTableProcedure
 org.apache.paimon.flink.procedure.MigrateFileProcedure
 org.apache.paimon.flink.procedure.RemoveOrphanFilesProcedure
 org.apache.paimon.flink.procedure.QueryServiceProcedure
-org.apache.paimon.flink.procedure.ExpireSnapshotsProcedure
+org.apache.paimon.flink.procedure.ExpireSnapshotsProcedure
```

Comment:
Do you mean to add procedures "create empty branch" & "create branch from snapshot"

Reply:
> Do you mean to add procedures "create empty branch" & "create branch from snapshot"

@schnappi17 Merged in the CreateBranchProcedure.